src/java/org/apache/cassandra/config/EncryptionOptions.java

```diff
@@ -67,6 +68,42 @@ public String description()
         }
     }
 
+    public enum ClientAuth
+    {
+        REQUIRED("true"),
+        NOT_REQUIRED("false"),
+        OPTIONAL("optional");
+        private final String value;
+        private static Map<String, ClientAuth> values = new HashMap<>();
+        static
+        {
+            for(final ClientAuth clientAuth: ClientAuth.values())
```

Comment:
NIT: Usage of final is discouraged in C* code style
```suggestion
            for (ClientAuth clientAuth : ClientAuth.values())
```